crates/holochain/src/conductor/api/api_external/admin_interface.rs

```diff
@@ -353,268 +309,3 @@ impl AdminInterfaceApi {
-        );
-
-        // with a property should install and produce a different hash
-        let json: serde_yaml::Value = serde_yaml::from_str("some prop: \"foo\"").unwrap();
-        let hash_payload = RegisterDnaPayload {
-            modifiers: DnaModifiersOpt::none().with_properties(YamlProperties::new(json.clone())),
-            source: DnaSource::Hash(dna_hash.clone()),
-        };
-        let install_response = admin_api
-            .handle_admin_request(AdminRequest::RegisterDna(Box::new(hash_payload)))
-            .await;
-        assert_matches!(
-            install_response,
-            AdminResponse::DnaRegistered(hash) if hash != dna_hash
-        );
-
-        // with a network seed should install and produce a different hash
-        let hash_payload = RegisterDnaPayload {
-            modifiers: DnaModifiersOpt::none()
-                .with_network_seed(String::from("12345678900000000000000")),
-            source: DnaSource::Hash(dna_hash.clone()),
-        };
-        let hash2_install_response = admin_api
-            .handle_admin_request(AdminRequest::RegisterDna(Box::new(hash_payload)))
-            .await;
-
-        let new_hash = if let AdminResponse::DnaRegistered(ref h) = hash2_install_response {
-            h.clone()
-        } else {
-            unreachable!()
-        };
-
-        assert_matches!(
-            hash2_install_response,
-            AdminResponse::DnaRegistered(hash) if hash != dna_hash
-        );
-
-        // from a path with a same network seed should return the already registered hash so it's idempotent
-        let path_payload = RegisterDnaPayload {
-            modifiers: DnaModifiersOpt::none()
-                .with_network_seed(String::from("12345678900000000000000")),
-            source: DnaSource::Path(dna_path.clone()),
-        };
-        let path2_install_response = admin_api
-            .handle_admin_request(AdminRequest::RegisterDna(Box::new(path_payload)))
-            .await;
-        assert_matches!(
-            path2_install_response,
-            AdminResponse::DnaRegistered(hash) if hash == new_hash
-        );
-
-        // from a path with different network seed should produce different hash
-        let path_payload = RegisterDnaPayload {
-            modifiers: DnaModifiersOpt::none().with_network_seed(String::from("foo")),
-            source: DnaSource::Path(dna_path),
-        };
-        let path3_install_response = admin_api
-            .handle_admin_request(AdminRequest::RegisterDna(Box::new(path_payload)))
-            .await;
-        assert_matches!(
-            path3_install_response,
-            AdminResponse::DnaRegistered(hash) if hash != dna_hash
-        );
-
-        tokio::time::timeout(std::time::Duration::from_secs(1), handle.shutdown())
-            .await
-            .ok();
-        Ok(())
-    }
-
-    // @todo fix test by using new InstallApp call
```

Comment:
Removed this commented out test here and added a small compensation in the client crate.